app/components/Analyst/Project/ProjectInformation/ProjectInformationForm.tsx

```diff
@@ -10,6 +10,66 @@ import { useCreateProjectInformationMutation } from 'schema/mutations/project/cr
 import ProjectTheme from 'components/Analyst/Project/ProjectTheme';
 import MetabaseLink from 'components/Analyst/Project/ProjectInformation/MetabaseLink';
 import Toast from 'components/Toast';
+import validateFormData from '@rjsf/core/dist/cjs/validate';
+import { Alert } from '@button-inc/bcgov-theme';
+
+const StyledAlert = styled(Alert)`
+  margin-bottom: 8px;
+  margin-top: 8px;
+`;
+
+export const displaySowUploadErrors = (err) => {
+  const { level: errorType, error: errorMessage } = err;
+  let title =
+    'An unknown error has occured while validating the Statement of Work data';
+  if (errorType?.includes('tab')) {
+    title = `There was an error importing the Statement of Work data at ${errorType}`;
+  }
+  if (errorType === 'summary') {
+    title =
+      'There was an error importing the Statement of Work data at the Summary tab';
+  }
+
+  if (errorType === 'database') {
+    title = 'An error occured when validating the Statement of Work data';
+  }
+
+  if (errorType === 'workbook') {
+    title =
+      'The Statement of Work sheet does not appear to contain the correct tabs.';
+  }
+  // for cell level errors
+  if (typeof errorMessage !== 'string') {
+    return errorMessage.map(({ error: message }) => {
+      return (
+        <StyledAlert
+          key={message}
+          variant="danger"
+          closable={false}
+          content={
+            <>
+              <div> {title}</div>
+              <div>{message}</div>
+            </>
+          }
+        />
+      );
+    });
+  }
+  return (
+    <StyledAlert
+      key={errorMessage}
+      variant="danger"
+      closable={false}
+      content={
+        <>
+          <div> {title}</div>
+          <p>{errorMessage}</p>
```

Comment:
This `<p>` 😅

Reply:
Ah! Sorry the test file I'm using was only rendering the divs. Thanks haha